validator/validator-main.protoascii

```diff
@@ -659,6 +659,23 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP metadata, name=amp-script-src
+tags: {
+  html_format: AMP
+  tag_name: "META"
+  spec_name: "meta name=amp-script-src"
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "content"
+    mandatory: true
```

Comment:
Just double-checking, the runtime will handle all values (for some reasonable definition of "handle" like "fail closed")?

Reply:
Not sure what you mean by "fail closed" but yes. :)